qiskit/primitives/backend_estimator_v2.py

```diff
@@ -254,8 +265,8 @@ def _postprocess_pub(
             for pauli, coeff in bc_obs[index].items():
                 expval, variance = expval_map[param_index, pauli]
                 evs[index] += expval * coeff
-                variances[index] += variance * coeff**2
-        stds = np.sqrt(variances / shots)
+                variances[index] += coeff * variance**0.5
+        stds = variances / np.sqrt(shots)
```

Comment:
Why do you suggest this change? It does not make sense to me to store `variance ** 0.5` to `variances`.

Reply:
Thank you for the details. I understand that the current implementation of stds is too small.
It might be nice to rename array `variances` because it stores square root of variances.
But since this method is short, I'm fine to leave the name as is.